test/helpers/fixture_set.rb

```diff
@@ -28,7 +28,14 @@ def app_name
     end
 
     def refute_resource_exists(type, name, beta: false)
-      client = beta ? v1beta1_kubeclient : kubeclient
+      client = if beta
+        v1beta1_kubeclient
+      elsif type == 'deployment'
```

Comment:
Should this include the other kinds that had to be upgraded: `StatefulSet, DaemonSet, ...`